formatter_filters/cloudevents/formatter_filter_test.go

```diff
@@ -279,6 +308,16 @@ func TestFormatterFilter_Process(t *testing.T) {
 			if tt.wantCloudEvent.ID == "" {
 				tt.wantCloudEvent.ID = gotCloudEvent.ID
 			}
+			if gotCloudEvent.SerializedHmac != "" {
+				hmac := tt.wantCloudEvent.SerializedHmac
+				tt.wantCloudEvent.SerializedHmac = ""
+				buf := &bytes.Buffer{}
+				enc := json.NewEncoder(buf)
+				enc.Encode(tt.wantCloudEvent)
+				require.NoError(err)
+				tt.wantCloudEvent.Serialized = base64.RawURLEncoding.EncodeToString(buf.Bytes())
+				tt.wantCloudEvent.SerializedHmac = hmac
```

Comment:
Not sure why you need to clear this above and then set it back to what it was here

Reply:
Oh never mind I see you encode `tt.wantCloudEvent`